lib/storage/metadata/mongoclient/MongoClientInterface.js

```diff
@@ -420,15 +420,35 @@ class MongoClientInterface {
              * code is the one related to mentionned issue.
              */
             if (err) {
-                if (!err.index || err.index !== 1
-                 || !err.code || err.code !== 11000) {
-                    log.error(
-                        'putObjectVerCase1: error putting object version',
-                    { error: err.errmsg });
+                if (err.code === 11000) {
+                    log.debug('putObjectVerCase1: error putting object version', {
+                        code: err.code,
+                        error: err.errmsg,
+                        isRetry: isRetry ? 'true' : 'false',
```

Comment:
or log the boolean directly with `isRetry,` (it should then be logged as a JSON boolean `true`, not `"true"`)

Reply:
i went with toString() :)

Reply:
in case of undefined, it will not be in the log, so to be consitent, i keep it the way i wrote it :)

Reply:
Or, to keep the type (not very important but might help with elasticsearch indexing for example)

```suggestion
                        isRetry: isRetry ? true : false,
```

Reply:
update for boolean